### ISSUE TITLE
mingw gets detected as java version

### ISSUE THREAD

**Issue by Lenni0451** [Col-E/Recaf-Launcher#29]
The launcher detects mingw64 as a valid java installation and displays it as Java 64.
mingw is in the following path: `C:\Users\User\AppData\Local\Programs\mingw64`

![grafik](https://github.com/user-attachments/assets/1f6c5903-6cc1-402c-b410-7cbe5fa03814)

**Comment by Col-E** [comment]
1. That's hilarious
2. Can you do `tree /F` in the `mingw` directory?

**Comment by Lenni0451** [comment]
I just downloaded mingw, unzipped it and put it into the `PATH` system variable.
After looking at the code it seems like the code is not checking if the java executable actually exists. I was able to add another version which does not exist by creating a `java128/bin` folder and adding it to the `PATH` variable.
![grafik](https://github.com/user-attachments/assets/dc8b6d60-05f2-4689-80e1-29e384a82abe)
![grafik](https://github.com/user-attachments/assets/f6375a3e-ab61-4514-98a4-486c35d62027)
